docs/public-networks/how-to/use-configuration-file/profile.md

```diff
@@ -1,26 +1,18 @@
 ---
+title: Use a profile 
 sidebar_position: 1
 ---
 
-# Use a profile
+# Use a profile 
 
-To help you get started quickly, Besu provides pre-configured profiles containing custom settings
-for some common use cases.
-You can load these profiles using the [`--profile`](../../reference/cli/options.md#profile) CLI option.
-For example:
+Load a profile using the [`--profile` CLI option](../../reference/cli/options.md#profile).
 
-```bash
-besu --config-file=config.toml --profile=staker
-```
-
-In this example, `config.toml` is the user-provided [configuration file](index.md), and `staker` is
-the pre-configured profile containing custom settings.
-
-Any configuration options explicitly set in the configuration file or command line will overwrite
-the same options set in the profile.
-See the [configuration order of precedence](index.md#configuration-order-of-precedence) for more information.
+:::note
+Run `./besu --help` to view all available profiles.
+:::
 
-You can use the following profiles:
+You can optionally use profiles to extend Besu's configuration. Combined with the [boilerplate 
+configuration](defaults.md), profiles simplify the process of applying viable defaults. Besu's pre-configured profiles optimize for the following use cases:
```

Comment:
Pretty sure this is the problem line - you might need a `../defaults.md`

Reply:
There are more linting issues come up unrelated to this PR's updates -- this issue became defunct with change of name and merging of the defaults into newly-named pg